examples/docs_snippets/docs_snippets/guides/components/index/34-dg-component-check-defs.txt

```diff
@@ -1,8 +1,8 @@
 dg check defs
 
 All components validated successfully.
+dagster definitions validate --log-level warning --log-format colored --workspace /tmp/workspace
```

Comment:
This is the only side effect of switching to loggers. For some reason this line (now emitted by the logging system rather than `print`) is now printed before the `INFO` lines below (also emitted by the logging system).